### PR TITLE
Abbr

### PR DESCRIPTION
**Pull request by orinevares**

### CODE REVIEW

File: components/abbreviations/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+title: Abbreviations
+status: Draft
+author: orinevares
+---
+
+![Status](https://img.shields.io/badge/Recommended-Draft-orange.svg)
+
+> Last Updated: May 30, 2019
+
+# Abbreviation
```
Comment (dlevineBC) [inline components/abbreviations/README.md]: Abbreviations & Acronyms

File: components/abbreviations/README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+An abbreviation is a shortened form of a word, phrase, or name where the abbreviation has not become part of the language. They are often used to reduce redundancy within a body of text.
+
+## Example
+
+<component-preview path="components/abbreviations/sample.html" height="100px" width="800px"> </component-preview>
+
+## Use This For:
+* Formal titles or names that occur frequently in your writing
```
Comment (dlevineBC) [inline components/abbreviations/README.md]: Providing the full name of an abbreviation or acronym

File: components/abbreviations/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+description: Abbreviations are shortened versions of a word or phrase.
+title: Abbreviations
```
Comment (dlevineBC) [inline components/abbreviations/README.md]: Abbreviations and Acronyms

File: components/abbreviations/README.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+status: Draft
+author: orinevares
+---
+
+![Status](https://img.shields.io/badge/Recommended-Draft-orange.svg)
+
+> Last Updated: May 30, 2019
```
Comment (dlevineBC) [inline components/abbreviations/README.md]: June 5, 2019

**Comment by orinevares** [comment]
@dlevineBC This seems to be stuck in review. Was there anything else you wanted changed?

**Pull request merged**